chainio/clients/avsregistry/writer.go

```diff
@@ -552,3 +552,37 @@ func (w *ChainWriter) SetSlashableStakeLookahead(
 	}
 	return receipt, nil
 }
+
+// This function creates a new Quorum from the old flow (without staking). The operator set params contains
+// the max operator count for that quorum and some churn options, the strategy params contains the specified
+// strategy for that quorum and its corresponding multiplier and the minimum stake is the minimum required
+// to the operator to be in the quorum. Returns the transaction receipt in case of success.
```

Comment:
Let's do something similar to #531 here.
```suggestion
// Creates a new quorum that tracks total delegated stake for operators.
// It receives the operator set parameters for the given quorum and the minimum stake required to register.
// Returns the transaction receipt in case of success.
// Note: This function does not work on M2 AVSs.
```

Reply:
Done in [Improve CreateTotalDelegatedStakeQuorum description](https://github.com/Layr-Labs/eigensdk-go/pull/529/commits/cc7b3da724ffd037cda195a89ad78c7c7721564e)